backend/app/adapter/routing/handle/search.go

```diff
@@ -54,7 +64,31 @@ func Search(
 			return
 		}
 
-		w.Write([]byte(fmt.Sprintf("%v", body)))
+		user := getUser(r, authenticator)
+		query := search.Query{
+			Query: body.Query,
+			User:  user,
+		}
+		filter, err := search.NewFilter(body.Filter.MaxResults, body.Filter.Resources, body.Filter.Orders)
```

Comment:
Is `MaxResults` a mandatory attribute? What happens if it not passed in search query? Will it be set to a defined default value or is it going to zero based on Go's default int value?

Reply:
you are correct! I will modify it now so we have a constant of max results in the case that it is not passed in the search query

Reply:
@rohithbalaji123 To me I think the correct the behavior should be when maxResult not specified, we return all.

Reply:
I think this part of business logic instead of API level thing.

Reply:
Good point!